src/app/amsterdam/bestuur-en-organisatie/college-van-burgemeester-en-wethouders/page.tsx

```diff
@@ -0,0 +1,130 @@
+'use client'
+
+import { AspectRatio, Breadcrumb, Card, Grid, Heading, PageHeading, Paragraph } from '@amsterdam/design-system-react'
+import NextImage, { type StaticImageData } from 'next/image'
+import NextLink from 'next/link'
+
+import municipalExecutive from '../../_assets/municipal-executive.jpg'
```

Comment:
Why import every image instead of using the source the persons array? Not sure about the performance implications here, but it would seem that leaving the downloading to the image component is more efficient.

Reply:
I’m not sure, either. @alimpens What would you do?

Reply:
Alright, that seems exactly what we’re doing here.